### PR TITLE
Fix tsdataset getitem method

### PR DESCRIPTION
**Pull request by iKintosh**

### CODE REVIEW

**Comment by Mr-Geekman** [comment]
Should we also add test for 3 index `__getitem__`?

**Comment by iKintosh** [comment]
> Should we also add test for 3 index `__getitem__`?

Added

**Comment by Mr-Geekman** [comment]
I think there can be a problem with understanding of `__getitem__` function. May be for user it is not obvious that we remove first rows with nan-s and how this indexing works. Can we add documentation to this method? However, I'm not sure that `__getitem__` method can be seen in documentation.

File: etna/datasets/tsdataset.py [hunk 1]
```diff
@@ -22,6 +22,29 @@ class TSDataset:
     """TSDataset is the main class to handle your time series data.
     It prepares the series for exploration analyzing, implements feature generation with Transforms
     and generation of future points.
+
+    Notes
+    -----
+    TSDataset supports custom indexing and slicing method.
+    It maybe done through these interface: TSDataset[timestamp, segment, column]
+    If at the start of the period dataset contains NaN those timestamp will be removed.
```
Comment (Mr-Geekman) [inline etna/datasets/tsdataset.py:30]: timestamp -> timestamps may be

**Review by Mr-Geekman — APPROVED** [review]

**Pull request merged**